ts/features/mvl/screens/MvlLoadingScreen.tsx

```diff
@@ -0,0 +1,43 @@
+import { View } from "native-base";
```

Comment:
Shouldn't we avoid `native-base` for new components?

Reply:
At the moment this is only applicable to textual components.
For others components like `<View spacer={true} extralarge={true} />` we don't have, atm, an alternative.
In general, I think we should understand for what we want to use `native-base` and when we like to avoid it!